nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/PutBigQuery.java

```diff
@@ -124,6 +127,15 @@ public class PutBigQuery extends AbstractBigQueryProcessor {
         .required(true)
         .build();
 
+    public static final PropertyDescriptor BIGQUERY_API_URL = new PropertyDescriptor.Builder()
+        .name("bigquery-api-url")
+        .displayName("BigQuery API URL")
+        .description("Overrides the default BigQuery URL.")
```

Comment:
Is it possible to add the actual default BigQuery URL to the description and/or when you would want to override it?

Reply:
Along these lines, perhaps it would be clearer to make this a required property, with the default value supplied.

Reply:
Good point! Then the changed logic stays the same and doesn't need the `if` clause

Reply:
I followed the same approach as in #7255 and #7172 to be consistent but I can definitely provide the default endpoint (at least in the description of the property).